web/src/ui_util.ts

```diff
@@ -75,3 +76,21 @@ export function parse_html(html: string): DocumentFragment {
     template.innerHTML = html;
     return template.content;
 }
+
+/*
+ * Handle permission denied to play audio by the browser.
+ * This can happen due to two reasons: user denied permission to play audio
+ * unconditionally and browser denying permission to play audio without
+ * any interactive trigger like a button. See
+ * https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/play for more details.
+ */
+export async function play_audio(elem: HTMLVideoElement): Promise<void> {
+    try {
+        await elem.play();
+    } catch (error) {
+        if (!(error instanceof DOMException)) {
+            throw error;
+        }
+        blueslip.error(`Unable to play audio. ${error.name}: ${error.message}`);
```

Comment:
Can you explain the thinking behind the error handling logic here? I think we possibly want something like a `blueslip.debug` level for the permission denied class of error, since those are presumably things behaving normally.

Reply:
Yeah, makes sense to use `debug` here. Updated!